package.json

```diff
@@ -59,34 +71,58 @@
     "prop-types": "^15.6.1"
   },
   "devDependencies": {
+    "@types/react": "^16.1.0",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.0",
+    "babel-eslint": "^8.2.2",
+    "babel-jest": "^22.4.3",
     "babel-plugin-external-helpers": "^6.22.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.13",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.24.1",
     "chokidar": "^1.6.1",
     "create-styled-element": "^0.4.0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
```

Comment:
Look at this: https://github.com/i18next/react-i18next/pull/414
Maybe use multiple versions of React for tests?

Reply:
Cool thanks, I wasn't aware of this helper.

Reply:
Going to wait for Enzyme to release a new version of the react-16 adapter to support the new Context API then I'll implement this.